docs/components/integration-framework/connectors/out-of-the-box-connectors/googledrive.md

```diff
@@ -0,0 +1,200 @@
+---
+id: googledrive
+title: Google Drive connector
+description: Create empty folders or documents from templates from your BPMN process.
+---
+
+The **Google Drive connector** allows you to create empty folders or documents from templates from your BPMN process.
+
+## Prerequisites
+
+To start working with the **Google Drive connector**, a relevant OAuth token must be configured and stored as a secret in the user's cluster. The token has to have permission to read and create a folder and/or files from a desired Google Drive instance. Follow the steps from the appendix to find out more about creating an OAuth token and giving relevant permissions.
+
+## Create a Google Drive connector task
+
+Currently, the **Google Drive connector** supports two flavors: create a new folder and create a file from a template.
+
+To use a **Google Drive connector** in your process, either change the type of existing task by clicking on it and using the wrench-shaped **Change type** context menu or create a new connector task by using the **Append connector** context menu. Follow our guide on using connectors to learn more.
+
+## Make your Google Drive connector for creating a folder executable
+
+To make the **Google Drive connector** executable, fill out the mandatory fields highlighted in red in the properties panel:
+
+![Google Drive connector new folder example](../img/connectors-googledrive-new-folder-filled.png)
+
+1. In the **Authentication** section, set the field **Google Drive Bearer Token** to your OAuth token, for example, `secrets.GDRIVE_OAUTH_TOKEN`.
+2. In the **Select Operation** section, set the field value **Operation Type** as **Create Folder**.
+3. In the **Operation Details** section, set the field **Folder name** as the desired name of a folder you wish to create. For example, `MyNewFolder`. Alternatively, you could use a FEEL expression.
+4. _(optional)_ In the **Operation Details** section, set the field **Parent folder ID** to the desired parent, inside which a new folder will be created. Please, keep in mind, that if not specified, a new folder will be created in the Google Drive root folder of a user, who owns the OAuth token.
+5. _(optional)_ In the **Operation Details** section, set the field **Additional properties or metadata**, set Google Drive compatible properties or metadata. This property requires FEEL input. Check the appendix for known values and limitations.
```

Comment:
"set the field **Additional properties or metadata** to..."?

I'm confused by "In the **Operation Details** section, set the field **Additional properties or metadata**, set Google Drive compatible properties or metadata" as it isn't a complete sentence. What do you mean here?

Reply:
What is meant here, there's a section called **Operation Details**. This section contains an input field **Additional properties or metadata**. In this section, user can input Google Drive compatible properties, e.g. _description_ of the folder.

Reply:
So do you mean...
"In the **Operation Details** section, you can set the **Additional properties or metadata** field to Google Drive compatible properties. For example, *description* of the folder."

Reply:
Yes, updated.